guides/tutorial/part-2/ember-data.md

```diff
@@ -329,27 +329,27 @@ import JSONAPISerializer from '@ember-data/serializer/json-api';
 
-Adding the `.json` extension is a bit less common, and doesn't have a declarative configuration API of its own. Instead, we will need to _override_ Ember Data's [`buildURL`](https://api.emberjs.com/ember-data/release/classes/JSONAPIAdapter/methods/buildURL?anchor=buildURL) method. Inside of `buildURL`, we will call `super.buildURL(...args)` to invoke the `JSONAPIAdapter` default implementation of `buildURL`. This will give us the URL that the adapter _would have built_, which would be something like `/api/rentals` and `/api/rentals/grand-old-mansion` after configuring the `namespace` above. All we have to do is to append `.json` to this URL and return it.
+Ajouter une extension `.json` est un peu moins répandu, il n'y a pas d'API de configuration déclarative propre pour le faire. Alors, nous avons besoin de surcharger la méthode [`buildURL`](https://api.emberjs.com/ember-data/release/classes/JSONAPIAdapter/methods/buildURL?anchor=buildURL) de Ember Data. Dans `buildURL` nous appelons `super.buildURL(...args)` pour invoquer son implémentation par défaut, celle de `JSONAPIAdapter`. Ça nous renvoie l'URL que l'adapteur "aurait construite", quelque chose comme `/api/rentals` et `/api/rentals/grand-old-mansion`, étant donné la configuration du `namespace` au-dessus. Tout ce que nous avons à faire, c'est ajouter `.json` à cette URL puis la retourner.
 
-Similarly, serializers are located at `app/serializers`. Adapters and serializers are always added together as a pair. We added an `application` adapter, so we also added a corresponding serializer to go with it as well. Since the JSON data returned by our server is JSON:API-compliant, the default [`JSONAPISerializer`](https://api.emberjs.com/ember-data/release/classes/JSONAPISerializer) work just fine for us without further customization.
+De la même manière, les sérialiseurs se situent dans `app/serializers`. L'adapteur et le sérialiseur sont toujours ajoutés par paire. Nous avons ajouté un adapteur `application`, donc nous avons aussi ajouté le sérialiseur correspondant. Puisque les données JSON retournées par notre serveur sont conformes à JSON:API, le sérialiseur par défaut [`JSONAPISerializer`](https://api.emberjs.com/ember-data/release/classes/JSONAPISerializer) fonctionne très bien pour nous, pas besoin de le personnaliser.
 
-With our adapter and serializer in place, all our tests should pass again.
+Avec notre adapteur et notre sérialiseur en place, nos tests devraient passer à nouveau.
 
-<img src="/images/tutorial/part-2/ember-data/pass-2@2x.png" alt="Once again, all the tests are passing again!" width="1024" height="1024">
+<img src="/images/tutorial/part-2/ember-data/pass-2@2x.png" alt="Une fois de plus, tous nos tests passent !" width="1024" height="1024">
 
-The UI works exactly the same as before as well, just with much less code!
+L'UI aussi fonctionne exactement comme avant, seulement avec moins de code&nbsp;!
```

Comment:
I will replace it with "interface utilisateur" without the English term because it's just one occurrence. In general, maybe we can go with _UI_ if the term is used a lot in the same file.